scripts/ci/strix_resolve_target_visibility.py

```
@@ -0,0 +1,294 @@
+    if any(marker in folded for marker in TRANSIENT_MARKERS):
+        return "transient"
+    return "unknown"
+
+
+def run_gh_visibility(
+    repository: str, timeout: float = DEFAULT_TIMEOUT_SECONDS
+) -> str:
+    """Return ``gh api`` stdout for ``repos/<repository>.private``."""
+    argv = ["gh", "api", f"repos/{repository}", "--jq", ".private"]
+    try:
+        completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)